### PR TITLE
Remove version from Windows Terminal;

### PR DESCRIPTION
**Pull request by Renegade-Master**
- windows/StartI2PdBrowser.bat
- windows_prebuilt/src/StartI2PdBrowser.bat
  - Uses Git to determine current version.
  - Displays the correct version in the startup window.

### CODE REVIEW

**Comment by r4sas** [comment]
I think we can just drop version from scripts, because release archives won't provide git information.

**Comment by Renegade-Master** [comment]
That is true. I was just getting a bit confused when I would update, and then be shown an older version. Made me go and double-check the scripts to make sure I had actually updated.

The indirection is the issue, not the version itself.
Please feel free to close this 🙂

File: windows/StartI2PdBrowser.bat [hunk 1]
```diff
@@ -41,12 +41,12 @@ exit /b 0
 rem ==========================================================================
 
 rem ==========================================================================
-rem ��������� EchoWithoutCrLf
-rem 
-rem %1 : ����� ��� ������.
+rem ��������� EchoWithoutCrLf
```
Comment (Renegade-Master) [inline windows/StartI2PdBrowser.bat:44]: Just saw this.

What is the encoding supposed to be for these files?

Comment (r4sas) [reply windows/StartI2PdBrowser.bat:44]: CP866

**Comment by r4sas** [comment]
Thank you.

**Pull request merged**